R/zchunk_L111.nonghg_en_R_S_T_Y.R

```diff
@@ -1,110 +1,303 @@
+    L111.nonghg_tg_R_en_Si_F_Yh.tmp1 %>%
+      rename(year = xyear) %>%
+      # Join in USA non-ghg emission factor data.  Keep NAs for now.
+      left_join(L111.nonghg_tgej_USA_en_Sepa_F_Yh.tmp1, by = c("Non.CO2", "EPA_agg_sector", "EPA_agg_fuel", "year"))  %>%
+      rename(emfact = value)  %>%
+      # Scale emission factors by energy sectors.
+      mutate(epa_emissions = energy * emfact) %>%
+      na.omit() %>%
+      left_join_keep_first_only(select(GCAM_sector_tech, EDGAR_agg_sector, sector, fuel),
+                                by = c("sector", "fuel")) ->
+      L111.nonghg_tg_R_en_Si_F_Yh.tmp1
+
+    # Create column of total EPA emissions by EDGAR sector and region
+    L111.nonghg_tg_R_en_Si_F_Yh.tmp1 %>%
+      group_by(GCAM_region_ID, Non.CO2, EDGAR_agg_sector, year) %>%
+      summarise(EPA_emissions = sum(epa_emissions)) ->
+      L111.nonghg_tg_R_en_Sedgar_Yh.tmp1
+
+    # Compute EDGAR emissions by region and sector.
+
+    # Add gas name and bind all dataframes together.
+    EDGAR_SO2$Non.CO2 <- "SO2"
+    EDGAR_CO$Non.CO2 <- "CO"
+    EDGAR_NOx$Non.CO2 <- "NOx"
+    EDGAR_NH3$Non.CO2 <- "NH3"
+
+    EDGAR_NMVOC %>%
+      mutate(Non.CO2 = "NMVOC") %>%
+      bind_rows(EDGAR_SO2, EDGAR_CO, EDGAR_NOx, ., EDGAR_NH3)  %>%
+      # Filter for EDGAR years (1971-2008) and 1970.
+      filter(year, year %in% emissions.EDGAR_YEARS_PLUS)  %>%
+      left_join(EDGAR_sector, by =c("IPCC", "IPCC_description"))  %>%
+      rename(IPCC_Annex = `IPCC-Annex`, World_Region = `World Region`) %>%
+      rename(EDGAR_agg_sector = agg_sector) %>%
+      #Convert from EDGAR iso to GCAM_region_ID, and change Romanian iso code.
+      standardize_iso(col = "ISO_A3") %>%
+      change_iso_code('rou', 'rom') %>%
+      left_join_keep_first_only(iso_GCAM_regID, by = "iso")  %>%
+      select(-country_name, -region_GCAM3)  ->
+      L111.EDGAR
+
+    # Create new table for international shipping & aviation emissions only and drop unnecessary columns.
+    L111.EDGAR %>%
+      filter(iso %in% c("sea","air")) %>%
+      select(-IPCC, -IPCC_description, -`World_Region`, -iso, -Name, -GCAM_region_ID, -`IPCC_Annex`) %>%
+      na.omit() ->
+      L111_EDGAR_intl
+
+    # Drop unnecessary columns, aggregate by region, and group by region, sector, NONGHG, and year.
+    L111.EDGAR %>%
+      select(-IPCC, -IPCC_description, -World_Region, -iso, -Name, -IPCC_Annex) %>%
+      na.omit() %>%
+      filter(year,year %in% emissions.EDGAR_YEARS) %>%
+      group_by(GCAM_region_ID, Non.CO2, EDGAR_agg_sector, year) %>%
+      summarize(value = sum(value))  ->
+      L111.EDGAR.tmp1
+
+    # Scale EPA emissions by tech to match EDGAR totals.
+
+    # First, compute scalers.
+    L111.nonghg_tg_R_en_Sedgar_Yh.tmp1 %>%
+      left_join(L111.EDGAR.tmp1, by = c("GCAM_region_ID", "Non.CO2", "EDGAR_agg_sector", "year"))  %>%
+      rename(EDGAR_emissions = value)  %>%
+      mutate(scaler = EDGAR_emissions / EPA_emissions / 1000.0) ->
```

Comment:
@bpbond @cahartin 
FMS and I cannot figure out this supposed unit conversion. Hypothetically, this line is tg/tg/1000.  So it would be unitless?  Farther down the code on line ~216? I divide by 1000.